backend/Tasque.Core.WebAPI/appsettings.json

```diff
@@ -23,7 +23,7 @@
     "IsSandboxMode": false
   },
   "SendGridOptions": {
-    "ApiKey":  ""
+    "ApiKey": "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"
```

Comment:
Better not to commit such parts as secret keys or connection strings, but this time - ok